toolkit/tools/pkg/imagecustomizerlib/customizepackages.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
There is a bit of overlap here with `TdnfInstall()` and `TdnfInstallWithProgress()`. They aren't really the same, but just wanted to make sure there wasn't anything we can reuse.

Reply:
Yeah, I needed some subtle changes for image customization. Hence, the code duplication.